src/ROOTReader.cc

```diff
@@ -7,15 +8,17 @@
 #include "podio/GenericParameters.h"
 #include "rootUtils.h"
 
+#include <TBranch.h>
+#include <TObjArray.h>
+#include <TObject.h>
+#include <algorithm>
+#include <functional>
+#include <optional>
+#include <stdexcept>
+#include <unordered_map>
```

Comment:
```suggestion
#include <TBranch.h>
#include <TObjArray.h>
#include <TObject.h>

#include <algorithm>
#include <functional>
#include <optional>
#include <stdexcept>
#include <unordered_map>
```
Additionally, the system headers should go to the bottom.